README.md

```diff
@@ -1 +1,87 @@
-# idl-repository
\ No newline at end of file
+# Interface Definition Language Repository (idl-repository)
+
+A repository and command line interface (CLI) for storing and fetching interface definitions in various formats. ([Protocol Buffers][protobuf], [Avro][avro], [OpenAPI][openapi], etc.)
+
+[protobuf]: https://developers.google.com/protocol-buffers/
+[avro]: https://avro.apache.org/
+[openapi]: https://swagger.io/docs/specification/about/
+
+## Quickstart
+
+### Initialize a project for the first time
+
+Initializing a project creates a new `idl.yaml` file with the project name and repository URL.
+
+```bash
+idl init my-project http://idl-repository.example.com
+```
+
+Read more about [`idl init`][idl-init].
+
+### Pull dependencies into your project
+
+Pull IDLs into your project.
+
+```bash
+idl pull
+```
+
+Read more about [`idl pull`][idl-pull].
+
+### Push project to the repository
+
+Push IDLs in your project to the configured repository.
+
+```bash
+idl push
+```
+
+Read more about [`idl push`][idl-push].
+
+### Documentation
+
+Read the full documentation for [`idl`][idl].
+
+Read the full documentation for the repository service, [`idl-repository`][idl-repository].
+
+[idl]: docs/idl/idl.md
+[idl-init]: docs/idl/idl_init.md
+[idl-pull]: docs/idl/idl_pull.md
+[idl-push]: docs/idl/idl_push.md
+[idl-repository]: docs/idl-repository/idl-repository.md
+
+## Building
+
+[![Travis](https://img.shields.io/travis/syncromatics/idl-repository.svg)](https://travis-ci.org/syncromatics/idl-repository)
+[![Docker Build Status](https://img.shields.io/docker/build/syncromatics/idl-repository.svg)](https://hub.docker.com/r/syncromatics/idl-repository/)
+
+Ensure you have your `GOPATH` configured properly. (Typically, you'll want to check this repo out to `$(go env GOPATH)/src/github.com/syncromatics/idl-repository`.) You'll also need Docker to build the Docker image for the repository.
```

Comment:
Since this is using go modules having it in the go path is no longer a requirement.

Reply:
Done